types/proposal.go

```diff
@@ -30,6 +30,7 @@ type Proposal struct {
 	BlockID   BlockID   `json:"block_id"`
 	Timestamp time.Time `json:"timestamp"`
 	Signature []byte    `json:"signature"`
+	Data      []byte    `json:"data"` // [peppermint] tx data
```

Comment:
Please, please, remove this, otherwise the structure of Proposal for your fork of `v0.38.x` might be incompatible with vanilla `v0.38.x`

Reply:
Got it!

Reply:
Resolved